### DISCUSSION TITLE
how to integrate in my HA?

### DISCUSSION THREAD

**Discussion by migius** [elax46/custom-brand-icons#148] [Q&A] [answered]
Hi, I'm new to HA world and honestly I don't know how to add your graet work to my HA, can you add a mini setup wiki?

**Comment by elax46** [comment, marked as answer]
> Hi, I'm new to HA world and honestly I don't know how to add your graet work to my HA, can you add a mini setup wiki?

The installation guide is [here ](https://github.com/elax46/custom-brand-icons#install) You still need to install first [hacs](https://hacs.xyz/)
